### PR TITLE
Interface for fillna. SQL portion will be separate.

### PR DESCRIPTION
**Pull request by kvnkho**

### CODE REVIEW

File: fugue/workflow/workflow.py
Comment (WangCHX) [inline fugue/workflow/workflow.py:535]: Nice.

**Review by WangCHX — APPROVED** [review]
good stuff. LGTM. need to wait for @goodwanghan 's approval.

File: fugue/execution/execution_engine.py
Comment (goodwanghan) [inline fugue/execution/execution_engine.py:373]: It's better to move dropna and fillna metadata to the last parameter

**Comment by goodwanghan** [comment]
Looks perfect. Thanks Kevin. Great work.

**Review by goodwanghan — APPROVED** [review]

**Pull request merged**
